docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -187,10 +187,10 @@ The default bundle supports the following minimum versions:
 <!-- #stable-snapshot -->
 
 - Node 12 (up from 8)
-- Chrome 84 (up from 49)
-- Edge 85 (up from 14)
+- Chrome 90 (up from 84)
+- Edge 91 (up from 85)
 - Firefox 78 (up from 52)
-- Safari 13 (macOS) and 12.2 (iOS) (up from 10)
+- Safari 14 (macOS) and 13.4 (iOS) (up from 13)
```

Comment:
Are we sure about this one? It would mean we can remove `doesSupportTouchActionNone()`, #27724.

Reply:
> Is this breaking change light enough to fly in a beta version?

Yep, since this was always the intention.

Reply:
> It would mean we can remove doesSupportTouchActionNone(), #27724.

Yeah, should've maybe done this in reverse.

Reply:
We stay on ios_saf 12.4 for two reasons:
1. iPhone 6 support
2. webpack 4 support (nullish coalescing and optional chaining are not supported by their parser version (acorn))